Cargo.toml

```diff
@@ -1,15 +1,16 @@
 [package]
 name = "vtebench"
-version = "0.1.2"
+version = "0.2.0"
+authors = ["Christian Duerr <contact@christianduerr.com>", "Joe Wilm <joe@jwilm.com>"]
 license = "Apache-2.0"
-authors = ["Joe Wilm <joe@jwilm.com>"]
 description = "Benchmark terminal emulators"
-documentation = "https://docs.rs/vtebench"
```

Comment:
Why was this removed?

Reply:
It's a binary and not a lib, there's no value in such thing.

Reply:
Oh, fair point. I forgot about that.